CI test fails: ImportError: cannot import name 'require_soundfile' from 'transformers.testing_utils'

CI test fails: https://github.com/huggingface/smolagents/actions/runs/16538930362/job/46777428127?pr=1604
> ImportError: cannot import name 'require_soundfile' from 'transformers.testing_utils'

```python
_____________________ ERROR collecting tests/test_types.py _____________________
ImportError while importing test module '/home/runner/work/smolagents/smolagents/tests/test_types.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_types.py:21: in <module>
    from transformers.testing_utils import (
E   ImportError: cannot import name 'require_soundfile' from 'transformers.testing_utils' (/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/transformers/testing_utils.py)
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -69,7 +69,7 @@ toolkit = [
 ]
 transformers = [
   "accelerate",
-  "transformers>=4.0.0",
+  "transformers>=4.0.0,<4.54.0",  # Hotfix for GH-1619
   "smolagents[torch]",
 ]
 vision = [

